src/mlpack/methods/random_forest/random_forest_impl.hpp

```diff
@@ -192,13 +196,14 @@ double RandomForest<
          const size_t minimumLeafSize,
          const double minimumGainSplit,
          const size_t maximumDepth,
-         DimensionSelectionType dimensionSelector)
+         DimensionSelectionType dimensionSelector,
+         bool warmStart)
 {
   // Pass off to Train().
   arma::rowvec weights; // Ignored by Train().
-  return Train<false, true>(dataset, datasetInfo, labels, numClasses, weights,
+  return Train<false, false>(dataset, datasetInfo, labels, numClasses, weights,
```

Comment:
Why change this to `false`?  This is an overload that has a `datasetInfo`.  Maybe a typo? :)

Reply:
Yes, my bad 😅. Somehow while copy pasting, I didn't realize that the template arguments were different. Thanks for pointing out :)